xarray/core/computation.py

```diff
@@ -1047,6 +1046,163 @@ def earth_mover_distance(first_samples,
         return apply_array_ufunc(func, *args, dask=dask)
 
 
+def cov(da_a, da_b, dim=None):
+    """Compute covariance between two DataArray objects along a shared dimension.
+
+    Parameters
+    ----------
+    arrays: DataArray (or Variable) objects
+        Arrays to compute.
+    dim : str, optional
```

Comment:
I think this can be a list too, but we can leave for another PR

Reply:
Yes, good point!